### PR TITLE
Share origin_gce install code across conformance jobs

### PR DESCRIPTION
**Pull request by smarterclayton**

### CODE REVIEW

File: sjb/config/test_cases/test_pull_request_origin_extended_conformance_k8s.yml [hunk 1]
```diff
@@ -1,15 +1,13 @@
 ---
-parent: 'common/test_cases/origin_release.yml'
+parent: 'test_cases/test_branch_origin_extended_conformance_gce.yml'
 overrides:
-  junit_analysis: False
```
Comment (stevekuznetsov) [inline sjb/config/test_cases/test_pull_request_origin_extended_conformance_k8s.yml]: why? being pushed to gubernator, right?

Comment (smarterclayton) [reply sjb/config/test_cases/test_pull_request_origin_extended_conformance_k8s.yml]: it's set in the base

Comment (smarterclayton) [reply sjb/config/test_cases/test_pull_request_origin_extended_conformance_k8s.yml]: Or do I also have to override in a descendant?

Comment (stevekuznetsov) [reply sjb/config/test_cases/test_pull_request_origin_extended_conformance_k8s.yml]: oh no sorry didn't see that it was in ancestor

**Pull request merged**